src/electron/index.ts

```diff
@@ -100,26 +98,12 @@ function createWindow(tunnelAtShutdown?: SerializableTunnel) {
     event.preventDefault();  // Prevent the app from exiting on the 'close' event.
     mainWindow.hide();
   };
-  mainWindow.on('minimize', minimizeWindowToTray);
```

Comment:
I think this is the right behavior (at least for Windows): when the user clicks minimize, the main window should hide and clicking the system tray icon should restore it.

Would it be worth casing it for linux as a workaround for the click event issue?

Reply:
Shouldn't we get the default system behavior if we don't override the event?
It would be nice to test this on Windows, but I don't have a way to do that.

Notice that the user can still close the window.

Reply:
I'll test on Windows and let you know if it works as expected.